File: classic/Code/LengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(const char *s) {
        int i=-1;
        int size=strlen(s);
        for(i=size-1;i>=0;i--){
            if(s[i]!=' '){
                break;
            }
        }
        if(i==-1){return 0;}
        int count=0;
        for(int j=i;j>=0;j--){
            if(s[j]==' '){break;}
            count++;
        }
        return count;
    }
};

File: classic/Code/LengthOfLastWord.java
public class Solution {
    public int lengthOfLastWord(String s) {
        int n=s.length();
        if(s.length()==0){
            return 0;
        }
        int counter=0;
        int index=s.length()-1;
        while(index>=0&&s.charAt(index)==' '){
            index--;
        }
        while(index>=0&&s.charAt(index)!=' '){
            counter++;
            index--;
        }
        return counter;
    }
}
